ipaserver/rpcserver.py

```diff
@@ -493,7 +493,7 @@ def marshal(self, result, error, _id=None,
 
     def unmarshal(self, data):
         try:
-            d = json.loads(data)
+            d = json.loads(ipautil.decode_json(data))
```

Comment:
`data` is read from `wsgi.input` and according to PEP 3333, reading from `wsgi.input` returns `bytes`, so we don't need to guess here and can always decode as UTF-8.